docs/membership/anonymous-mode.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Anonymous Mode
+description: Learn more about Anonymous Mode
+---
+
+<Tip>
+  [Upgrade here](https://app.codecrafters.io/pay) to get unlimited content
+  access.
+</Tip>
+
+A CodeCrafters membership grants you access to Anonymous Mode. It allows you to hide your GitHub username and avatar on CodeCrafters.
+
+## What is Anonymous Mode?
+
+Anonymous Mode masks the GitHub username and avatar on your CodeCrafters profile.
+
+Your code examples and profile will still be public but under a random username and profile picture.
```

Comment:
This can be hard to visualize - I'd add a picture somewhere. You can enable this on your account to get a screenshot.